src/ralph/lib/mixins/admin.py

```diff
@@ -0,0 +1,9 @@
+class MemorizeBeforeStateMixin(object):
+
+    def save_model(self, request, obj, form, change):
+        """
+        Given a model instance save it to the database.
```

Comment:
https://github.com/django/django/blob/master/django/contrib/admin/options.py#L999-L1003

I will describe class and remove this irrelevant docstring.